README.md

```diff
@@ -46,14 +46,27 @@
 
 # OpenTelemetry Collector Contrib
 
-This is a repository for OpenTelemetry Collector contributions that are not
-part of the [core
-distribution](https://github.com/open-telemetry/opentelemetry-collector) of the
-Collector. Typically, these contributions are vendor specific
-receivers/exporters and/or components that are only useful to a relatively
-small number of users.
-
-> Please note that this repository and its releases are a superset of the core repository.
+This is a repository for OpenTelemetry Collector components that are not suitable for the  [core repository](https://github.com/open-telemetry/opentelemetry-collector) of the collector. Some of these components are part of the core distribution, such as the Jaeger and Prometheus ones, but most of the components here are only available as part of the "contrib" distribution. Users of the OpenTelemetry Collector are also encouraged to build their own custom distributions with the [OpenTelemetry Collector Builder](https://github.com/open-telemetry/opentelemetry-collector/tree/main/cmd/builder), using the components they need from the core repository, the contrib repository, and possibly third-party or internal repositories.
+
+The official distributions, core and contrib, are available as part of the [opentelemetry-collector-releases](https://github.com/open-telemetry/opentelemetry-collector-releases) repository.
+
+Each component has its own stability and support levels, as defined in the following sections.
+
+## Stability levels
```

Comment:
Should we have a way to expose this on the zpages or with a special command on the otelcol binary?

Reply:
Good call. I think @esnible added an option to print out all the components of the collector upon error, perhaps a code similar to that would be good to have here. I'll create another issue to track this.

Reply:
Created #9518 and #9519.